x-pack/plugins/infra/public/pages/metrics/metric_detail/hooks/use_metrics_time.ts

```diff
@@ -84,11 +84,16 @@ export const useMetricsTime = () => {
   };
 };
 
-export const MetricsTimeInputRT = rt.type({
+const timeRt = rt.type({
   from: rt.union([rt.string, rt.number]),
   to: rt.union([rt.string, rt.number]),
+});
+
+const intervalRt = rt.partial({
```

Comment:
Why is the interval partial? Do we have a case when we don't have an interval?

Reply:
I was just reviewing this part because it smelled bad somehow, and I figured out that on the Hosts page the date range picker has not a default interval defined in the `useUrlState` default value. I'll update this part to provide a default value and make it mandatory again :)